src/charactersheet/services/common/hotkeys_service.js

```diff
@@ -11,7 +11,7 @@ export var HotkeysService = {
         if (keypressIsInBody) {
             var metaKey = HotkeysService._determineMetakey(event);
 
-            var alphaNumericKey = String.fromCharCode(event.which);
+            var alphaNumericKey = event.key;
```

Comment:
What's driving this change?

Reply:
The `String.fromCharCode()` method cannot detect the `escape` key. 

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/fromCharCode

Reply:
ah ok